.github/workflows/kvrocks.yaml

```diff
@@ -206,7 +206,7 @@ jobs:
         run: |
           export ${{ matrix.runtime_env_vars }}
           ./x.py test tcl build --cli-path $HOME/local/bin/redis-cli --dont-clean
-          if [[ ! -z "${{ matrix.with_openssl }}" ]] && [[ "${{ matrix.os }}" == ubuntu* ]]; then
+          if [[ ! -z ${{ matrix.with_openssl }} ]] && [[ "${{ matrix.os }}" == ubuntu* ]]; then
```

Comment:
I think if `matrix.with_openssl` does not exist, it will be replaced to `[[ ! -z ]]` rather than `[[ ! -z "" ]]`, hence maybe it will fail due to syntax error.

Reply:
BTW  `[[ -n "something" ]]` is another approach to check if it is not empty : )

Reply:
Thanks @PragmaTwice